packages/cypress-commands/src/index.js

```diff
@@ -1,6 +1,21 @@
+// will automatically create the commands
+import './commands/fillAndSubmitLoginForm'
+import './commands/find'
+import './commands/get'
+import './commands/login'
+import './commands/stubWithFixture'
+import './commands/visitWhenStubbed'
+
 // helpers
 export { dataTestNameToSelector } from './helper/dataTestNameToSelector'
 export { parseSelectorWithDataTest } from './helper/parseSelectorWithDataTest'
 
-// commands
-export { registerCommands } from './commands/registerCommands'
+// setup helpers
+export { enableAutoLogin } from './setups/enableAutoLogin'
+
+// backward compatibility
+export const registerCommands = () => {
+    cy.log('The usage of `registerCommands` has been depricated')
+    cy.log('This function does nothing anymore and can be removed')
```

Comment:
`... doesn't do anything anymore ...` ?
`This function does nothing and can be removed` ?

Reply:
I agree.

Reply:
with which of the three alternatives?

Reply:
Sorry, I misinterpreted things... Get it now though... I think it's important to mention all three aspects actually, but I think we can provide that info in one sentence perhaps:
```
The usage of `registerCommands` has been deprecated and is now a no-op, because commands are registered automatically.
```